### PR TITLE
Make :ls/:buffers invoke ibuffer

### PR DESCRIPTION
**Pull request by rgrinberg**
Only when evil +everywhere and ibuffer are enabled

### CODE REVIEW

File: modules/emacs/ibuffer/config.el [hunk 1]
```diff
@@ -68,6 +68,9 @@
                default-directory)
            default-directory)))))
 
+  (when (featurep! :editor evil +everywhere)
+    (evil-ex-define-cmd "buffers" #'ibuffer))
```
Comment (hlissner) [inline modules/emacs/ibuffer/config.el:72]: Could you move this to modules/editor/evil/+commands.el, in a `(when (featurep! :emacs ibuffer) ...)` check? This rules out any race conflicts caused by ibuffer loading before evil-ex is loaded [due to how we lazy load of Evil's ex commands](https://github.com/hlissner/doom-emacs/blob/develop/modules/editor/evil/config.el#L195-L197).

Comment (hlissner) [reply modules/emacs/ibuffer/config.el:72]: Oh, and I don't think the `+everywhere` check is necessary. Extra ex commands aren't anywhere near as imposing as all the extra evilified keybinds that comes with that flag.

Comment (rgrinberg) [reply modules/emacs/ibuffer/config.el]: Done.

**Comment by hlissner** [comment]
Looks good to me! Thanks again!

**Pull request merged**